scipy/stats/stats.py

```diff
@@ -7465,14 +7535,29 @@ def ranksums(x, y):
     ----------
     x,y : array_like
         The data from the two samples.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+          * 'two-sided':
+                The alternative hypothesis says that the location shift
+                between the two distributions is not equal to 0.
+          * 'less':
+                The alternative hypothesis says that the location shift
+                between the two distributions is less than 0.
```

Comment:
Which way is the location shift defined? `x - y` or `y - x`? I think "location shift" makes some additional assumptions about the test, so I'm going to suggest using the idea of "stochastically greater/less".

Reply:
> Which way is the location shift defined? `x - y` or `y - x`?

It's `x - y`. 

> I think "location shift" makes some additional assumptions about the test, so I'm going to suggest using the idea of "stochastically greater/less".

Sure, I will change it to that.

Reply:
Oh, I see you just changed it. Thanks!

Reply:
Is this new wording OK? Do you agree that it corresponds with the [more general formulation](https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test#:~:text=Under%20the%20general)?

Reply:
> Is this new wording OK? Do you agree that it corresponds with the [more general formulation](https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test#:~:text=Under%20the%20general)?

Yes, I think the new description looks correct.